lib/package.json

```diff
@@ -99,18 +99,15 @@
     "@types/react": "^16.9.35",
     "@types/react-dom": "^16.9.8",
     "@types/react-transition-group": "^4.2.4",
-    "@typescript-eslint/eslint-plugin": "^1.6.0",
-    "@typescript-eslint/parser": "^1.6.0",
     "babel-loader": "^8.1.0",
     "babel-plugin-optimize-clsx": "^2.6.1",
     "babel-plugin-react-remove-properties": "^0.3.0",
     "codecov": "^3.7.1",
     "cross-env": "^7.0.2",
     "date-fns": "^2.12.0",
+    "dayjs": "^1.8.27",
     "enzyme": "^3.11.0",
     "enzyme-adapter-react-16": "^1.15.2",
-    "eslint-plugin-react": "^7.19.0",
-    "eslint-plugin-react-hooks": "^4.0.5",
```

Comment:
These are used in the `.eslintrc` in the root. Therefore they should be declared in the root package.json.